dev/com.ibm.ws.microprofile.config.1.2_fat/test-applications/converterApp/src/com/ibm/ws/microprofile/config12/converter/priority/web/ConverterPriorityServlet.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.config12.converter.priority.web;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import com.ibm.ws.microprofile.config12.converter.priority.beans.ConverterPriorityBean;
+
+import componenttest.app.FATServlet;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+
+@SuppressWarnings("serial")
+@WebServlet("/ConverterPriorityServlet")
+public class ConverterPriorityServlet extends FATServlet {
+
+    @Inject
+    ConverterPriorityBean bean;
+
+    @Test
+    @Mode(TestMode.LITE)
```

Comment:
don't need to annotate for `@Mode(TestMode.LITE)`, that is the default value if no `@Mode` annotation is present